.travis.yml

```diff
@@ -0,0 +1,47 @@
+---
+sudo: required
+services:
+  - docker
+language: python
+python:
+  - "2.6"
+  - "2.7"
+env:
+  global:
+    - OS_NAME=centos
+  matrix:
+    - OS_MAJOR_VERSION=6
+    - OS_MAJOR_VERSION=7
+before_install:
+  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+  - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+  - sudo apt update
+  - sudo apt install -y docker-ce
+  - docker --version
+install:
+  - wget http://archive.ubuntu.com/ubuntu/pool/universe/r/rpmlint/rpmlint_1.7-1_all.deb
+  - sudo dpkg -i rpmlint_1.7-1_all.deb || sudo apt install -f -y
+  - gem install mdl
+  - if [[  $TRAVIS_PYTHON_VERSION == 2.6 ]]; then pip install 'flake8<3.0.0'; fi
+  - if [[  $TRAVIS_PYTHON_VERSION == 2.7 ]]; then pip install flake8 flake8-import-order hacking; fi
+script:
+  # First syntax, linting and pre-test stages
+  - mdl -s relaxed README.md
+  - rpmlint glite-info-update-endpoints.spec
+  - flake8 bin/glite-info-update-endpoints
+  # Then, start the build container
+  - docker run --name build_container -d -ti -v $PWD:/tmp -w /tmp $OS_NAME:$OS_MAJOR_VERSION /bin/bash -c 'while true ; do sleep 1000 ; done'
+  # Now, install prerequisites for the build
+  - docker exec -i -w /root build_container yum install -y rpm-build make rsync
+  # Eventually build the rpm
+  - docker exec -i -w /tmp build_container make rpm
+  # and install it
+  - docker exec -i -w /tmp/build/RPMS/noarch build_container bash -c "rpm -Uvh glite-info-update-endpoints*.el${OS_MAJOR_VERSION}.noarch.rpm"
+deploy:
+  provider: releases
+  api_key:
+    secure: l5zpstoCnh06FIwNJ5VXuTF+KUkGBsNV/Jo3t0eRqI2DShfppdi8hYZa6bEjW2B0V73EMXQKuxpeBpo/gVWqWK5c7QBW1Gv/1malwaI8po1SBOKzQx0BaKOxVZwB6oBE1b1HsX8ywGFj44MgbIndd7bZJSRBePHgrbr5UXs6A/dVgyI4q0PDhqV72vzXK6E1Mme8U0ekYToPpMD8m3F9TbrnGrbiGEJC3sl36QlTVCkAEFpxxPMxAP4n3wvyZFDcTWkz5PcfppHStDPOYUcngf1HfUfiXaJ6KTADIQXFxAKDpX7hUlqzs1RtLyZg+be6l/KCnNkqwR/aDaka2NW285m56enx4flntIFCDmRUdipADnFjH7NXvWudhG55l8lnYm1FVxbDskAagAJbzQgaLIHT874sRrvYLMDb0bE4yat7NN9dMROImP1W3KrQ6qIu5ZRI6BA4kA7YP1+ij5sAwRhxA8hN9Y59NjpdJTt8GOQRJw9bk5EbKWBf5eu6n+c5ma/1avG7elJXsA79ijH+CWFcK+hQrTuDmqavdlRpJyYA+FB0hyBZKTETWEEYnpOkZ3De3Ak2Ng/pNdSTN+mf7uy7lSzX61io9CwMk5eflZSMBI3esYa5dxkU6yLFBN9jcbExj54jrfyFWIPHaDBF5DvBJ4ajR6Oat4hg9kotgxs=
+  file_glob: true
+  file: build/RPMS/noarch/glite-info-update-endpoints*.el${OS_MAJOR_VERSION}.noarch.rpm
```

Comment:
beautiful

Reply:
I'm still unsure how it will act as we have 4 build jobs, (python 2.6 and 2.7 on CentOS 6 and 7) so we will have two times two conflicting/identical packages.

Reply:
maybe you can [name the jobs in the matrix](https://docs.travis-ci.com/user/customizing-the-build/#naming-jobs-within-matrices), build rpms with a name crafted from that var, and then copy it to the current name when the build passes?